libindi/drivers/telescope/telescope_script.cpp

```diff
@@ -79,7 +78,11 @@ const char * ScopeScript::getDefaultName() {
 bool ScopeScript::initProperties() {
   INDI::Telescope::initProperties();
   
+#ifdef OSX_EMBEDED_MODE
   IUFillText(&ScriptsT[0], "FOLDER", "Folder", "/usr/local/share/indi/scripts");
+#else
+  IUFillText(&ScriptsT[0], "FOLDER", "Folder", "/usr/share/indi/scripts");
```

Comment:
That's right, any script itself should be prepared by the user, they definitely have persistent nature and /var is not a right choice. But this is just default value and I suppose it will be changed by the user anyway.